Content.Server/EntityEffects/Effects/PlantMetabolism/PlantDestroySeeds.cs

```diff
@@ -0,0 +1,45 @@
+﻿using Content.Server.Botany.Components;
+using Content.Server.Botany.Systems;
+using Content.Shared.EntityEffects;
+using Robust.Shared.Prototypes;
+using Content.Shared.Popups;
+
+using Content.Shared.Popups;
```

Comment:
You are importing the same namespace twice. Using an IDE is really helpful since it warns you about stuff like this.

Reply:
I'm using VSCode with effectively no extensions, so... oops. Thanks for the catch!